pkg/utils/clusterinfo/fetcher.go

```diff
@@ -376,3 +376,38 @@ func getPDNodesHealth(pdEndPoint string, httpClient *http.Client) (map[string]st
 	}
 	return memberHealth, nil
 }
+
+func GetAlertCount(info *AlertManagerInfo, httpClient *http.Client) (int, error) {
+	apiAddress := fmt.Sprintf("http://%s:%d/api/v2/alerts", info.IP, info.Port)
+	resp, err := httpClient.Get(apiAddress)
+	if err != nil {
+		return 0, err
+	}
+	defer resp.Body.Close()
+	data, err := ioutil.ReadAll(resp.Body)
+
+	if err != nil {
+		return 0, err
+	}
+
+	var alerts []struct{}
+
+	err = json.Unmarshal(data, &alerts)
+	if err != nil {
+		return 0, err
+	}
+
+	return len(alerts), nil
+}
+
+func GetAlertCountByAddress(address string, httpClient *http.Client) (int, error) {
+	ip, port, err := parseHostAndPortFromAddress(address)
+	if err != nil {
+		return 0, err
+	}
+	return GetAlertCount(&AlertManagerInfo{
```

Comment:
why binaryPath need to be specified when getting alert count?

Reply:
Removed, PTAL again.